voidcrew/modules/cloning/code/computer_cloning.dm

```diff
@@ -0,0 +1,526 @@
+#define AUTOCLONING_MINIMAL_LEVEL 3
+
+/datum/data/record
+	name = "record"
+	var/list/fields = list()
+
+/obj/machinery/computer/cloning
+	name = "cloning console"
+	desc = "Used to clone people and manage DNA."
+	icon_screen = "dna"
+	icon_keyboard = "med_key"
+	circuit = /obj/item/circuitboard/computer/cloning
+	req_access = list(ACCESS_GENETICS) //for modifying records
+	light_color = LIGHT_COLOR_BLUE
+
+	///List of all user records we have stored on the computer.
+	var/list/datum/data/record/records = list()
+
+	///The disk inserted into the console, holding cloning data for patients that we can read/write from/to.
+	var/obj/item/disk/data/diskette
+	///The linked machine that holds the bodies we can scan.
+	var/obj/machinery/dna_scannernew/scanner
```

Comment:
``scanner`` is never used/set by anything.

Reply:
I think its used by the pod the computer gets connected to the dna_scannernew